plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -403,8 +405,6 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
           "extract build environment information as JSON, write into specified file, and exit")
          ("force-all-checks", bpo::bool_switch()->default_value(false),
           "do not skip any validation checks while replaying blocks (useful for replaying blocks from untrusted source)")
-         ("disable-replay-opts", bpo::bool_switch()->default_value(false),
```

Comment:
Will removing the command line option cause an error if it is still provided? Probably better than leaving an option doing nothing though.

Reply:
Everything in the `cfg` options is allowed on the command line or in the `config.ini`. Everything in the `cli` is only allowed on the command line.

Reply:
Oh, thanks!